source/_reference/basics.coffee

```diff
@@ -126,6 +126,13 @@ module.exports = exports =
 
                    See each method's doc for its id.
                    """
+      errors: [
+        key: "invalid-method"
+        http: "404"
+        description: """
+                     An invalid method id is sent over websockets.
```

Comment:
Given that we're already in context, and to be closer to the language used for other errors, I'd simply say `The given method id is invalid.`